File: src/resources/templates/Generator.java
import java.util.*;

public class Generator {
    public static void main(String[] args) {
        Random random = new Random();
        
        // Generate random test case
        // Example: generate random array
        int n = random.nextInt(10) + 1;
        
        System.out.println(n);
        for (int i = 0; i < n; i++) {
            System.out.print(random.nextInt(100) + 1);
            if (i < n - 1) System.out.print(" ");
        }
        System.out.println();
    }
}

File: src/app/resources/templates/generator.cpp
#include <iostream>
#include <random>
#include <chrono>
using namespace std;

int main() {
    // Seed random number generator
    mt19937 rng(chrono::steady_clock::now().time_since_epoch().count());
    
    // Generate random test case
    // Example: generate random array
    int n = uniform_int_distribution<int>(1, 10)(rng);
    
    cout << n << endl;
    for (int i = 0; i < n; i++) {
        cout << uniform_int_distribution<int>(1, 100)(rng);
        if (i < n - 1) cout << " ";
    }
    cout << endl;
    
    return 0;
}